tests/compute/entry-point-uniform-params.slang

```diff
@@ -0,0 +1,51 @@
+// entry-point-uniform-params.slang
+
+// Confirm that `uniform` parameters on
+// entry points are allowed, and work as expected.
+
+//TEST(compute):COMPARE_COMPUTE_EX:-slang -compute
+//TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -dx12
+//TEST(compute, vulkan):COMPARE_COMPUTE_EX:-vk -compute
+
+struct Signs
+{
+	int a;
+}
+
+struct Stuff
+{
+	int b;
```

Comment:
It'll be good if one of these parameters is a resource parameter.

Reply:
Note that the actual entry point has the parameter: `uniform RWStructuredBuffer<int> outputBuffer` for the test output buffer, so we do indeed have a resource there.

There is still a lot that can/should be done about better coverage of all the corner cases, though.